draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -368,10 +386,12 @@
             <t>
                 If the offerer inserts the SDP 'setup' attribute with an 'actpass' or 'passive' attribute value, the
                 offerer MUST be prepared to receive a DTLS ClientHello message (if a new DTLS association
-                is established by the answerer) from the answerer before the offerer receives the SDP answer.
+                is established by the answerer) from the answerer before the offerer receives the SDP answer. In such
+                case the offerer MAY cache the ClientHello message, but the offerer MUST NOT initiate a DTLS

```

Comment:
Just to be completely clear why this thing worries me, let me present an attack scenario:

1. Attacker knows the IP and port range for a communication service which allows early unverified media
2. Attacker sprays that range with unsolicited ClientHello messages
3. If attacker hits the end point at the right time, end point will respond with ServerHello
4. Attacker completes the handshake and start playing some sort of offensive or disturbing media to this end point
5. Callers in the organization end having an interesting user experience

This works even better if this is a corporate network behind NAT since a lot of media will be coming from a small number of public IPs.

Based on this, I was always a strong supporter of not playing the media until the other end point is verified.